libs/ledger-live-common/src/families/stellar/bridge/mock.ts

```diff
@@ -43,6 +43,10 @@ const createTransaction = (): Transaction => ({
   memoValue: null,
   memoType: null,
   useAllAmount: false,
+  operationType: "payment",
```

Comment:
Is `bridge/mock.ts` file needed? It doesn't look like it is used.

Reply:
It can be useful if we are doing screenshot comparaison test. I suggest to keep it for the moment